### ISSUE TITLE
[Bug]: TypeError: kill() missing 1 required positional argument: 'self'

### ISSUE THREAD

**Issue by ClownOfMadness** [camenduru/stable-diffusion-webui-colab#327]
### What happened?

![image](https://user-images.githubusercontent.com/52707645/229179789-ab095b5d-0d0a-415c-9de2-c5b77f4b3415.png)

I keep getting this error when i run the colab for the first time, everytime
I was using the stable version of the Meina_V8 model colab

### Console logs

```Shell
Traceback (most recent call last):
  File "/content/stable-diffusion-webui/launch.py", line 362, in <module>
    start()
  File "/content/stable-diffusion-webui/launch.py", line 356, in start
    webui.webui()
  File "/content/stable-diffusion-webui/webui.py", line 240, in webui
    app.add_middleware(GZipMiddleware, minimum_size=1000)
  File "/usr/local/lib/python3.9/dist-packages/starlette/applications.py", line 139, in add_middleware
    raise RuntimeError("Cannot add middleware after an application has started")
RuntimeError: Cannot add middleware after an application has started
Error in atexit._run_exitfuncs:
TypeError: kill() missing 1 required positional argument: 'self'
```

### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits

### Are you using the latest version of the colab notebook?

- [X] Yes, i am using the latest version from the repo

### Which colab notebook did you use?

Miena_V8

### Which repo branch did the colab you use comes from ?

stable

### Which tunnel did you use ?

--multiple

### What browsers do you use to access the WebUI ?

Mozilla Firefox

**Comment by Nicholls01** [comment]
seems somehow the WEBAPI version is wrongly installed to 0.95, try this fix:
find "!sed -i -e 's/fastapi==0.90.1/fastapi==0.89.1/g' /content/stable-diffusion-webui/requirements_versions.txt" in run.ipynb, and replace it with
"#!sed -i -e 's/fastapi==0.90.1/fastapi==0.89.1/g' /content/stable-diffusion-webui/requirements_versions.txt
!pip install --upgrade --force-reinstall fastapi==0.90.1"

**Comment by ClownOfMadness** [comment]
> seems somehow the WEBAPI version is wrongly installed to 0.95, try this fix: find "!sed -i -e 's/fastapi==0.90.1/fastapi==0.89.1/g' /content/stable-diffusion-webui/requirements_versions.txt" in run.ipynb, and replace it with "#!sed -i -e 's/fastapi==0.90.1/fastapi==0.89.1/g' /content/stable-diffusion-webui/requirements_versions.txt !pip install --upgrade --force-reinstall fastapi==0.90.1"

Works perfectly now, thank you!

**Comment by camenduru** [comment]
Hi @ClownOfMadness 👋 thanks ❤ fixed
